.eslintrc.js

```diff
@@ -1,25 +1,42 @@
 module.exports = {
   extends: 'airbnb',
+  env: { browser: true },
   rules: {
+    // ESLint ----------------------------------
+    'arrow-parens': ['error', 'always'],
```

Comment:
I think it's easier to read and understand

`var f = (arg) => arg + 1`

than

`var f = arg => arg + 1`

as far as declaring functions go